docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -41,8 +41,26 @@ The ``<participant>`` element has two attributes defined: ``profile_name`` and `
 DomainParticipant configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``<participant>`` element has as child element ``<rtps>`` under which all the DomainParticipant configuration
-options belong.
+The ``<participant>`` element has two child elements: ``<domain_id>`` and ``<rtps>``.
```

Comment:
If `<domain_id>` is not supported, it should be fixed here and refer only to `<domainId>` which is the supported one